src/registers/model_specific.rs

```diff
@@ -141,7 +141,7 @@ mod x86_64 {
             {
                 let low = value as u32;
                 let high = (value >> 32) as u32;
-                llvm_asm!("wrmsr" :: "{ecx}" (self.0), "{eax}" (low), "{edx}" (high) : "memory" : "volatile" );
+                asm!("rdmsr", in("ecx") self.0, in("eax") low, in("edx") high, options(nostack))
```

Comment:
This should be `wrmsr` instead of `rdmsr`.

Reply:
Correct, it should. I think it was a typo, I copy pasted a lot of that stuff and forgot to change it